index.php

```diff
@@ -1,9 +1,17 @@
 <?php
 
+require_once 'functions.php';
+require_once 'connectDB.php';
 
+$db = connectDB();
+$wines = getWines($db);
 
-
-
+$displayWines = '';
```

Comment:
you can delete this line as $displayWines gets defined on line 11, i think..

Reply:
Great thanks! took it out and it all still worked fine